src/main.cpp

```diff
@@ -65,6 +65,7 @@ ros::NodeHandle nh;
 #endif
 
 bool received_gait_instruction_response;
+bool send_gait_message;
```

Comment:
Change the name to `gait_message_send` for clarity, this sounds like a function

Reply:
Good point